sections/conclusion/conclusion-chapter.tex

```diff
@@ -1 +1,8 @@
 \section{Conclusion}\label{sec:conclusion}
+Throughout this paper, we have looked into various metrics used for evaluating recommender systems. What we found was that there is no consistent set of metrics used for evaluation, but the most commonly used are NDCG and Recall.
+We found that utilizing bipartite graphs is a popular and intuitive way to handle information in the system, but in the investigated metrics, they are often outperformed by traditional methods such as SVD.
+The cause of this is speculative but may be because methods like SVD make use of explicit rating information, where the investigated graph-based methods use implicit ratings.
+\\\\
+From the knowledge acquired throughout this paper, we propose that instead of only looking into context, as information about the interaction, it could be beneficial to expand the model to include side-information about the items or users in addition to the context.
```

Comment:
```suggestion
From the knowledge acquired throughout this paper, we propose that instead of only looking into context as information about the interaction, it could be beneficial to expand the model to include side-information about the items or users in addition to the context.
```